File: Spring_Boot/MicroServices/SB-DMS-Project/src/main/java/com/dms/model/Department.java
package com.dms.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor

@Entity
@Table(name = "dept")
public class Department {

	@Id
	@Column(name = "dno")
	private int depNo;

	@Column(name = "dname", length = 20)
	private String depName;

	public int getDepNo() {
		return depNo;
	}

	public void setDepNo(int depNo) {
		this.depNo = depNo;
	}

	public String getDepName() {
		return depName;
	}

	public void setDepName(String depName) {
		this.depName = depName;
	}

}

File: Micro_Project/Spring Boot/React/Individual/ems-app/src/model/Department.js
class Department {
    depNo;
    depName;
    constructor() {
       this.depNo=0;
       this.depName="";
    }

    setDepNo(depNo){
       this.depNo=depNo;
    }
    setDepName(depName){
       this.depName=depName;
    }
}
export default Department;